iconic/UserProfile/views.py

```diff
@@ -56,3 +56,90 @@ def get(self, request):
         resume = Resume.objects.get(user_id=request.user.id)
         data = CustomResumeSerializer(resume).data
         return Response(data)
+
+#View for changing and adding a profession
+class ProfessionsEdit(APIView):
+    
+    # Method for getting profession by id
+    def get(self, request):
+        profession = Professions.objects.get(id=request.data['id'])
+        serializer = ProfessionsSerializer(profession)
+        return Response(serializer.data)
```

Comment:
I want the status of the response. Look at the code from the updated dev as an example